docs/redux/redux-async.md

```diff
@@ -115,17 +115,16 @@ import {Component} from 'react';
 import {connect} from 'react-redux';
```

Comment:
The `LS2Request Example` seems not to use `redux-thunk` and still uses `connect`.
Would you please update the example to use `redux-thunk` and `useDispatch`?
Also, I found our pattern-ls2request sample (https://github.com/enactjs/samples/tree/master/ui/pattern-ls2request/src) seems old style. Would you please update it as we did in sandstone/pattern-virtuallist-preserving-focus sample (https://github.com/enactjs/samples/tree/master/sandstone/pattern-virtuallist-preserving-focus/src )?

Reply:
We use `applyMiddleware` from `redux-thunk` to create a thunk action. I will update the sample to use `useDispatch` .

Reply:
Oh.. I miss worded. not `redux-thunk` but `redux-toolkit. sorry.